app/controllers/application_controller.rb

```diff
@@ -1,14 +1,14 @@
 class ApplicationController < ActionController::Base
-	protect_from_forgery with: :exception
-	before_action :authenticate_user!
-	before_action :configure_permitted_parameters, if: :devise_controller?
+	 protect_from_forgery with: :exception
```

Comment:
2 space in all the files
please run rubocop and resolve all the comments before pushing the code